documentation/source/users/rmg/running.rst

```diff
@@ -20,83 +20,17 @@ Run with CPU profiling::
+    python rmg.py input.py -n <Max number of processes allowed>
 
-    #!/bin/bash
-
-    #SBATCH -p debug
-    #SBATCH -J jobname
-    #SBATCH -n 24
-
-    Processes=12
-    RMG_WS=/path/to/RMG/parent/folder
-    export PYTHONPATH=$PYTHONPATH:$RMG_WS/RMG-Py/
-
-    source activate rmg_env
-
-    python $RMG_WS/RMG-Py/rmg.py -n $Processes input.py
-
-    source deactivate
-
-Running RMG in parallel with SGE
---------------------------------
-
-RMG has the option to use multiple processes on one node for reaction generation and on-the-fly Quantum Mechanics Thermodynamic Property (QMTP) calculation. Here is an example submission script for an RMG-Py job with a SGE scheduler.
-
-The job reserves 24 tasks on a single node, but uses only 12 processes in parallel during
-the RMG-Py simulation.
-
-Make sure that:
-
--  the queue named ``debug`` exists on your SGE scheduler.
--  you modify the path to the parent folder of the RMG-Py installation
-   folder.
--  you have an anaconda environment named ``rmg_env`` that contains
-   RMG-Py's dependencies.
--  the working directory from which you launched the job
-   contains the RMG-Py input file ``input.py``.
-
-.. code:: bash
-
-    #! /bin/bash
-
-    #$ -l debug
-    #$ -N jobname
-    #$ -pe singlenode 24
-
-    Processes=12
-    RMG_WS=/path/to/RMG/parent/folder
-    export PYTHONPATH=$PYTHONPATH:$RMG_WS/RMG-Py/
-
-    source activate rmg_env
-
-    python $RMG_WS/RMG-Py/rmg.py -n $Processes input.py
-
-    source deactivate
+We recommend you make a job-specific directory for each RMG simulation. Some jobs can take quite a while to complete, so we also recommend using a job scheduler if working in an linux environment. 
 
 
-Details on the implementation
+Details on the multiprocessing implementation
 --------------------------------
 
 Currently, multiprocessing is implemented for reaction generation and the generation of QMfiles when using the QMTP option to compute thermodynamic properties of species. The processes are spawned and closed within each function. The number of processes is determined based on the ratio of currently available RAM and currently used RAM. The user can input the maximum number of allowed processes from the command line. For each reaction generation or QMTP call the number of processes will be the minimum value of either the number of allowed processes due to user input or the value obtained by the RAM ratio. The RAM limitation is employed, because multiprocessing is forking the base process and the memory limit (SWAP + RAM) might be exceeded when using too many processors for a base process large in memory.
 
 In python 3.4 new forking contexts 'spawn' and 'forkserver' are available. These methods will create new processes which share nothing or limited state with the parent and all memory passing is explicit. Once RMG is transferred to python 3 it is recommended to use the spawn or forkserver forking context to potentially allow for an increased number of processes.
```

Comment:
Now that we're on Python 3, is this paragraph still relevant? Or should it be updated to say this will be implemented in the future?

Reply:
@kspieks and I were planning to work on this today afternoon. By then we should have a better idea if it works. I would like to keep it as is until we know more.